src/datumaro/experimental/fields/images.py

```diff
@@ -303,3 +303,55 @@ def image_callable_field(format: str = "RGB", semantic: Semantic = Semantic.Defa
         ImageCallableField instance configured with the given parameters
     """
     return ImageCallableField(semantic=semantic, format=format)
+
+
+@dataclass(frozen=True)
+class ImageIdField(Field):
```

Comment:
I suggest to make this field more generic - `IdentifierField` - so that users can represent any kind of identifier with it, not only image ids.